pinot-transport/src/main/java/com/linkedin/pinot/transport/scattergather/ScatterGatherImpl.java

```diff
@@ -426,11 +426,13 @@ private void runInternal() {
       KeyedFuture<ServerInstance, NettyClientConnection> keyedFuture = null;
       boolean gotConnection = false;
       boolean error = true;
+      long timeRemaining = _timeoutMS - (System.currentTimeMillis() - _startTime);
+      long timeWaited = 0;
```

Comment:
Rename these to reflect time unit, ie something like connectionWaitTimeInMillis.